datalad/metadata/search.py

```diff
@@ -183,8 +180,8 @@ def __call__(match,
             else:
                 raise
 
-        cache_dir = opj(cfg.obtain('datalad.locations.cache'), 'metadata')
-        mcache_fname = opj(cache_dir, ds.id)
+        cache_dir = opj(get_git_dir(ds.path), 'datalad', 'cache')
```

Comment:
I kinda liked that feature of centralized cache while playing with installing/uninstalling `///` but yeah -- probably it would make more sense to keep it local -- easier to clean up etc